docs/management/admin/blocklist.asciidoc

```diff
@@ -16,7 +16,7 @@ The blocklist is not intended to broadly block benign applications for non-secur
 
 By default, a blocklist entry is recognized globally across all hosts running {elastic-defend}. If you have a https://www.elastic.co/pricing[Platinum or Enterprise subscription], you can also assign a blocklist entry to specific {elastic-defend} integration policies, which blocks the process only on hosts assigned to that policy.
 
-. Go to **Manage** -> **Blocklist**.
+. Find **Blocklist** in the main menu or use the {kibana-ref}/introduction.html#kibana-navigation-search[global search field].
```

Comment:
Flagging this because we ultimately [decided](https://elastic.slack.com/archives/C07LSV17Z6K/p1730127908063179?thread_ts=1729792951.947889&cid=C07LSV17Z6K) to use `navigation` over `main` since `navigation` is more generic. (Obv this edit applies to all of the updated areas below.)

```suggestion
. Find **Blocklist** in the navigation menu or use the {kibana-ref}/introduction.html#kibana-navigation-search[global search field].
```

Reply:
Thanks, will update the other instances in the next commit